account_receipt_print/views/account_move_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_move_form_receipt" model="ir.ui.view">
+        <field name="name">account.move.receipt</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <button name="button_draft" position="after">
+                <button
+                    name="print_receipt"
+                    type="object"
+                    string="Print"
```

Comment:
Why do you prefer adding an explicit button rather than show the report in the Print drop-down menu?
I think can confuse the user.

Reply:
It's a long story. :sweat_smile: 
Anyway, you are right, see https://github.com/PNLUG/account-invoicing/pull/1

Reply:
Done.